crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -79,9 +79,13 @@ spec:
                     priorityClassName:
                       description: If specified, indicates the pod's priority. If not specified, the pod priority will be default or zero if there is no default.
                       type: string
-                    runAsRoot:
-                      description: RunAsRoot allows admins to indicate that they wish to run the container inside the CatalogSource pod in a privileged mode as root.  This should only be enabled when running older catalog images which could not be run as non-root.
-                      type: boolean
+                    securityContextConfig:
+                      description: 'SecurityContextConfig allows users to configure a grpc pod''s pod.spec.securityContext and pod.spec.container[*].securityContext to meet expected Pod Security Admission(PSA) standards. By default, the grpc pod is configured to run in "restricted" mode. If set to "legacy" mode, the grpc pod''s pod.spec.securityContext and pod.spec.container[*].securityContext will not be set, and the pod will only run in PSA "privileged" namespaces. The "legacy" mode should be used only with older catalog images, which could not be run in restricted mode, More information about PSA can be found here: https://kubernetes.io/docs/concepts/security/pod-security-admission/'
```

Comment:
don't mean to throw a spanner in the works, but can it also run under 'baseline'?

Reply:
Yea looks like you're right. It can in fact run in Baseline. Updating the doc.

Reply:
@perdasilva done.